src/onedrive.rs

```diff
@@ -62,11 +62,15 @@ impl OneDrive {
     /// # Panics
     /// It panics if the underlying `reqwest::Client` cannot be created.
     pub fn new(access_token: impl Into<String>, drive: impl Into<DriveLocation>) -> Self {
+        #[cfg(not(target_arch = "wasm32"))]
         let client = Client::builder()
             .redirect(reqwest::redirect::Policy::none())
             .gzip(true)
             .build()
             .unwrap();
+        #[cfg(target_arch = "wasm32")]
+        let client = Client::builder().build().unwrap();
```

Comment:
Why? Do you mean `gzip` or `redirect` is unsupported on wasm32?

`redirect` must not be disabled because `get_item_download_url` requires it to correctly function. The documentation of `new_with_client` already mentioned this.

Reply:
Yep `reqwest` does not have `redirect` or `gzip` for `wasm32` target.

I propose for `wasm` we:
- Disable `get_item_download_url`
- Add `get_item_content`, which follows redirects and return a `Vec<u8>`

What do you think?

Reply:
> Yep `reqwest` does not have `redirect` or `gzip` for `wasm32` target.

Are there target-related difficulties about implementing them? Or is it just `reqwest` *have not yet* implemented them? It seems [`ClientBuilder` on wasm32](https://docs.rs/reqwest/0.12.4/wasm32-unknown-unknown/reqwest/struct.ClientBuilder.html) is pretty incomplete and have only few methods available.


> * Disable `get_item_download_url`
> * Add `get_item_content`, which follows redirects and return a `Vec<u8>`

I'm not a fan of providing different (and inconsistent) APIs on different targets.